components/TextSourceSans.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Text, StyleSheet } from 'react-native';
+import { FONTS, SIZES } from '../theme/fonts';
+
```

Comment:
actions complaint about SIZES

Reply:
ah thanks for the hint!

Out of curiosity, where do you see gh actions @phillimorland?

Reply:
I was wondering if you didn't see that, so when I click on 'add your review' and check through the code in that view, actions has comments where it's sad about things, would be good if this was included in out pre commit lint though, i'll check the config

Reply:
ah yes, true..there are comments. and also true that this should be caught by our linting.